src/main/java/eu/europeana/metis/sandbox/config/ConcurrencyJdbcConfig.java

```diff
@@ -0,0 +1,24 @@
+package eu.europeana.metis.sandbox.config;
+
+import javax.sql.DataSource;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.integration.jdbc.lock.DefaultLockRepository;
+import org.springframework.integration.jdbc.lock.JdbcLockRegistry;
+import org.springframework.integration.jdbc.lock.LockRepository;
+
+@Configuration
+public class ConcurrencyJdbcConfig {
```

Comment:
Should we call this LockRepositoryJdbcConfig?
Makes more sense given the name of the .sql files.

Reply:
true. done.